README.md

```diff
@@ -1,40 +1,131 @@
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
+
 # openui5-basic-template-app
-This is a basic app template which you can use as a starting point to build OpenUI5 apps.
+> [OpenUI5](https://github.com/SAP/openui5) basic template app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
```

Comment:
Please add this sentence after the first line:
"You can use it as a starting point to build custom OpenUI5 apps."